first-uppercase.js

```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję firstUppercase(), która przyjmie zdanie i zamieni pierwszą literę każdego
+// słowa na wielką.
+
+function firstUppercase(string) {
+    const words = string.split(" ");
+
+    for (let i = 0; i < words.length; i++) {
```

Comment:
It works but it's a perfect task for the `Array.map` method. Please read about it and use it instead of this `for` loop`.